### PR TITLE
Problem: Hare uses older Consul version 1.7.8

### PR DESCRIPTION
**Pull request by mssawant**
Solution:
Update hare requirement for Consul version to 1.9.1.

Signed-off-by: Mandar Sawant <mandar.sawant@seagate.com>

### CODE REVIEW

File: hare.spec [hunk 1]
```diff
@@ -45,7 +45,7 @@ BuildRequires: python36-devel
 BuildRequires: python36-pip
 BuildRequires: python36-setuptools
 
-Requires: consul = 1.7.8
+Requires: consul = 1.9.1
```
Comment (suvratjoshi) [inline hare.spec:48]: Also change needed in :
File README.md
Line 59 : sudo yum -y install consul-1.7.8

Comment (mssawant) [reply hare.spec:48]: Ah, excellent catch! thanks.

Comment (mssawant) [reply hare.spec:48]: Done.

**Review by knekrasov — APPROVED** [review]

**Comment by andriytk** [comment]
Few questions:
1) Why 1.9.1 if there is already 1.9.2?
2) Why not to try the more stable 1.8.x branch? Why to risk with 1.9? Is there some new feature there that we need?

**Comment by mssawant** [comment]
>Few questions:

Why 1.9.1 if there is already 1.9.2?
Why not to try the more stable 1.8.x branch? Why to risk with 1.9? Is there some new feature there that we need?

No, Just tried with the latest available Consul version, I found 1.9.1, looks like its1.9.2 now, but did some basic testing with 1.9.1,  bootstrapping which worked. So went ahead with it. I did not check 1.8.x. I think we need to run more tests to see if everything works fine. Probably cQA can do that or I can try to run some more failure test cases next week.

**Comment by mssawant** [comment]
@andriytk, tested bootstrapping, node failure, node join did not see any consul specific issue.

File: hare.spec (hunk 1 above)
Comment (knekrasov) [reply hare.spec:48]: @mssawant please have a look at #1475. Perhaps that approach is even better?

**Review by andriytk — APPROVED** [review]
Further improvement is continued at #1475.

**Pull request merged**